universal-application-tool-0.0.1/conf/messages.en-US

```diff
@@ -45,3 +46,4 @@ validation.stateRequired=Please enter state.
 validation.zipcodeRequired=Please enter valid ZIP code.
 validation.invalidZipcode=Please enter valid 5-digit ZIP code.
 validation.noPoBox=Please enter a valid address. We do not accept PO Boxes.
+validation.entityNameRequired=Please enter a value for each {0}.
```

Comment:
(non-blocking): Why not add the es-US version of this one?

Reply:
done!